index.js

```diff
@@ -45,17 +45,33 @@ module.exports = opts => {
 	const headers = opts.verbose ? verboseHeaders : defaultHeaders;
 
 	return pify(childProcess.execFile)('tasklist', args)
-		.then(stdout => neatCsv(stdout, {headers}))
-		.then(data => data.map(task => {
-			// Normalize task props
-			task.pid = Number(task.pid);
-			task.sessionNumber = Number(task.sessionNumber);
-			task.memUsage = Number(task.memUsage.replace(/[^\d]/g, '')) * 1024;
-
-			if (opts.verbose) {
-				task.cpuTime = sec(task.cpuTime);
-			}
-
-			return task;
-		}));
+		// INFO: No tasks are running which match the specified criteria.
+		// See: https://github.com/sindresorhus/tasklist/issues/9
+		.then(stdout => stdout.startsWith('INFO:') ?
+			[] :
+			neatCsv(stdout, {headers})
+		)
+		.then(data =>
+			new Promise((resolve, reject) =>
+				// eslint-disable-next-line array-callback-return
+				resolve(data.map(task => {
+					try {
+						// Normalize task props
+						task.pid = Number(task.pid);
+						task.sessionNumber = Number(task.sessionNumber);
+						task.memUsage = Number(task.memUsage.replace(/[^\d]/g, '')) * 1024;
+
+						if (opts.verbose) {
+							task.cpuTime = sec(task.cpuTime);
+						}
+
+						return task;
+					} catch (err) {
+						// Handle parsing errors gracefully
+						reject(err);
+					}
+				})
+			)
+		)
```

Comment:
This part is not strictly related to #9. It is a safety measure in case any of the input returned by `tasklist` is not as we expect it to be, for whatever reason. E.g. it could be that `tasklist` does not output the `INFO:` part on Chinese system, or perhaps it returns a differently formatted `cpuTime` on an Arabic system, or perhaps the `memUsage` is not a number but some string in older Windows versions, or whatever. My point is that we do parsing there, and since neither of us know the source code of `tasklist`, we cannot be confident that it doesn't break in weird edge cases. That's why I prefer to have this extra try-catch that rejects the Promise on errors, since a Promise should never, in any case, throw but always reject.